src/renderer/shared/ui-entities/AccountExplorer/AccountExplorers.tsx

```diff
@@ -1,14 +1,14 @@
 import { type PropsWithChildren, memo } from 'react';
 
-import { type AccountId, type Chain } from '@/shared/core';
+import { type AccountId, type Address, type Chain } from '@/shared/core';
 import { useI18n } from '@/shared/i18n';
 import { copyToClipboard, getAccountExplorer, toAddress } from '@/shared/lib/utils';
 import { ExplorerLink, FootnoteText, HelpText, IconButton, Separator } from '@/shared/ui';
 import { Box, Popover } from '@/shared/ui-kit';
 import { Hash } from '../Hash/Hash';
 
 type Props = PropsWithChildren<{
-  accountId: AccountId;
+  accountId: Address | AccountId;
```

Comment:
Actually we don't care what to pass `address` or `accountId`, few lines below we convert whatever we passed to `formattedAddress` using `toAddress` which works well with both data types. So the ability to pass `Address | AccountId` gives us more freedom

Reply:
It's important for API shape and readability, revert this change please. Otherwise rename component to `AddressExplorers` and change it semantic meaning